### ISSUE TITLE
High severity vulnerabilities in dependencies

### ISSUE THREAD

**Issue by henrijs** [postcss/postcss-custom-properties#235]
This package depends on `postcss-values-parser` that depends on `url-regex` that has high severity vulnerability.

Upstream issue https://github.com/shellscape/postcss-values-parser/issues/130. Please see https://npmjs.com/advisories/1550.

**Comment by Semigradsky** [comment]
I don't see `url-regex` in dependencies:
```
$ yarn why url-regex
yarn why v1.22.5
[1/4] Why do we have the module "url-regex"...?
[2/4] Initialising dependency graph...
[3/4] Finding dependency...
error We couldn't find a match!
Done in 0.27s.
```

**Comment by Semigradsky** [comment]
@henrijs upgrade your `postcss-values-parser` to 10+ version. It was fixed in #228
